site/docs/voice/bxml/startStream.mdx

```diff
@@ -0,0 +1,419 @@
+---
+id: startStream
+title: Start Stream
+slug: /voice/bxml/startStream
+description: A general overview of Bandwidth's StartStream BXML Verb
+keywords:
+  - bandwidth
+  - voice
+  - bxml
+  - start
+  - streaming
+hide_title: false
+image: ../../static/img/bandwidth-logo.png
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The StartStream verb allows a segment of a call to be sent off to another destination for additional processing.
+
+When used on a call, audio from one or both sides (tracks) of the call will be sent to the specified destination. The stream will continue until the call ends or the [`<StopStream>`][1] verb is used. A total of 4 concurrent track streams are allowed on a call. A `<StartStream>` request that uses `both` tracks will count as 2 concurrent streams.
+A call has only two tracks, named after the direction of the media from Programmable Voice perspective:
+- `inbound`: media received by Programmable Voice from the call executing the BXML;
+- `outbound`: media sent by Programmable Voice to the call executing the BXML.
+
+Note that this has no correlation to the direction of the call itself. For example, if either an inbound or outbound call is being streamed and executes a `<SpeakSentence>`, the `inbound` track will be the callee's audio and the `outbound` track will be the text-to-speech audio.
+## Text Content
+
+There is no text content available to be set for the `<StartStream>` verb.
+
+## Attributes
+
+| Attribute          | Description                                                                                                                                                                                                                        |
+|:-------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| name               | (optional) A name to refer to this stream by. Used when sending [`<StopStream>`][1]. If not provided, it will default to the generated stream id as sent in [`Media Stream Started`][2] webhook.                                             |
```

Comment:
```suggestion
| name               | (optional) A name to refer to this stream by. Used when sending [`<StopStream>`][1]. If not provided, it will default to the generated stream id as sent in the [`Media Stream Started`][2] webhook.                                             |
```